tsconfig.json

```diff
@@ -0,0 +1,25 @@
+{
+  "compilerOptions": {
+    "module": "commonjs",
+    "noImplicitAny": true,
+    "removeComments": true,
+    "preserveConstEnums": true,
+    "sourceMap": true,
+    "outDir": "dist",
```

Comment:
Electron builder already uses this folder when compiling to distributeables  so we may need to change this here

Reply:
I think this is fine. Electron builder seems to use release